python/sanlock.c

```diff
@@ -124,44 +124,26 @@ __parse_resource(PyObject *obj, struct sanlk_resource **res_ret)
     res->num_disks = num_disks;
```

Comment:
This patch runs fine with py3 in my tree, the issue is probably another patch.

Reply:
i re-based it to an earlier point in my branch, to get rid of some py3 helpers.